Sources/MapboxDirections/RouteResponse.swift

```diff
@@ -1,12 +1,12 @@
 import Foundation
 
-struct RouteResponse {
-    var code: String?
-    var message: String?
-    var error: String?
-    let uuid: String?
-    let routes: [Route]?
-    let waypoints: [Waypoint]?
+public struct RouteResponse {
+    public var code: String?
+    public var message: String?
+    public var error: String?
+    public let uuid: String?
+    public let routes: [Route]?
+    public let waypoints: [Waypoint]?
```

Comment:
This change requires us to address all of #391. Are you sure we should do so before integrating the navigation SDK with the first round of Swifitification in this library?